configs/coco_model.yaml

```diff
@@ -105,7 +96,7 @@ trainer:
   n_sanity_val_steps: 1
   profiler: null
   verbose: True
-  batch_size: 4
+  batch_size: 8
   accumulate_grad_batches: 1
   epochs: &epochs 200
   n_workers: 8
```

Comment:
Can we double check if `train_metrics_interval` parameter is even used anywhere? Because I believe we don't actually call a function where this is used. If that is the case then we should remove this config parameter (and the function)